packages/breadcrumbs/src/styled/StyledBreadcrumbItem.ts

```diff
@@ -6,23 +6,36 @@
  */
 
 import styled from 'styled-components';
-import classNames from 'classnames';
-import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
-import BreadcrumbStyles from '@zendeskgarden/css-breadcrumbs';
+import { getColor } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'breadcrumbs.item';
 
-export interface IStyledBreadcrumbItemProps {
-  isCurrent?: boolean;
-}
-
-export const StyledBreadcrumbItem = styled.li.attrs<IStyledBreadcrumbItemProps>(props => ({
+export const StyledBreadcrumbItem = styled.li.attrs(() => ({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: classNames(BreadcrumbStyles['c-breadcrumb__item'], {
-    // State
-    [BreadcrumbStyles['is-current']]: props.isCurrent
-  })
+  'data-garden-version': PACKAGE_VERSION
 }))<IStyledBreadcrumbItemProps>`
-  ${props => retrieveComponentStyles(COMPONENT_ID, props)};
+  ${props => `
+    line-height: ${props.theme.lineHeights.md};
+    color: ${props.isCurrent ? getColor(props.theme.colors.neutralHue, 600) : 'inherit'}
+    ${
+      props.isCurrent === undefined || props.isCurrent === false
+        ? `::after {
+        transform:  ${props.theme.rtl ? 'rotate(180deg)' : 'none'};
+        display: inline-block;
+        margin: 0 ${props.theme.space.xs};
+        background: no-repeat 50% / 1em url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='16' color='%2368737D'%3E%3Cpath fill='none' stroke='currentColor' stroke-linecap='round' d='M4.5 15l6.6-6.6c.2-.2.2-.5 0-.7L4.5 1'/%3E%3C/svg%3E");
```

Comment:
I copied over the chevron we have now in production and bumped up the size to 16 x 16. 

This was the most straight forward approach, but I suspect there are much better ways to do this. I also know that chevron icons exists in the [svg-icons](https://garden.zendesk.com/svg-icons/16px/) project, which might be useful here.

Reply:
You should be using the styling seen [here](https://garden.zendesk.com/css-components/breadcrumbs/). We got this in recently so that you would have the right designs to work off of.